README.md

```diff
@@ -25,7 +23,7 @@ Therefore, we recommend improving this model prior to deployment in the
 real-world.
 
 The data set used in this project is the “Online Shoppers Purchasing
-Intention” dataset provided by the [Gözalan
+Intention” dataset provided by the [GÃ¶zalan
```

Comment:
invalid unicode

Reply:
Hmm I swear I had fixed this last week. It seems to reset (again probably the formatter). I will fix it